central/image/service/service_impl.go

```diff
@@ -366,7 +366,9 @@ func (s *serviceImpl) GetImageVulnerabilitiesInternal(ctx context.Context, reque
 
 func (s *serviceImpl) acquireScanSemaphore() error {
 	if err := s.internalScanSemaphore.Acquire(concurrency.AsContext(concurrency.Timeout(maxSemaphoreWaitTime)), 1); err != nil {
-		s, err := status.New(codes.Unavailable, err.Error()).WithDetails(&v1.ScanImageInternalResponseDetails_TooManyParallelScans{})
+		// Aborted indicates the operation was aborted, typically due to a concurrency
+		// issues.  Clients should retry by default on Aborted.
+		s, err := status.New(codes.Aborted, err.Error()).WithDetails(&v1.ScanImageInternalResponseDetails_TooManyParallelScans{})
```

Comment:
I'm wondering if `RESOURCE_EXHAUSTED` makes more sense: https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Reply:
Why do you think we should use it instead?

Reply:
I was thinking that because we want to stick with a certain amount of concurrency, so reaching that limit "exhausts the resources". However, I'm actually not sure if this indicates that the user can try again later. `UNAVAILABLE` certainly does (based on the information in the link referenced above), but we can't use that I guess. Let's do `ABORTED`, then